src/main/resources/db/changelog/alert/5.3.0/changelog-user-management.xml

```diff
@@ -10,4 +10,11 @@
             CALL REMOVE_ALL_PERMISSIONS('ALERT_USER','component_authentication', 'GLOBAL');
         </sql>
     </changeSet>
+    <changeSet author="psantos" id="2019-12-18-07-31-16-844">
+        <addColumn schemaName="ALERT" tableName="USERS">
+            <column name="EXTERNAL" type="BOOLEAN" defaultValueBoolean="false">
```

Comment:
Won't all the users (Except the ones we created) be external at this point?

Reply:
I will add the column to 5.2.0 therefore this change can be omitted from this PR.

Reply:
In 5.2.0 the send of the authentication event is commented out.  So only in 5.3.0 will we sync external users.  Therefore this changeset is correct for 5.3.0 only